data/pages-content/terms.md

```diff
@@ -171,56 +162,54 @@ Organizers and/or third parties.
+article which is of a dangerous, explosive, hazardous or objectionable nature;
+
+2. _In-person participants_ will not try to gain unauthorised access to the
    Venue and only use the entrances and exits to the Venue, designated as such
    by the Venue;
 3. _For in-person participants_, no act or thing is done inside, outside or in
-   the vicinity of Venue, which may, in Organizer’s reasonable opinion, be or
+   the vicinity of Venue, which may, in Organiser’s reasonable opinion, be or
    become a nuisance to the attendees, the Venue, its other customers or any
    premises or buildings in its vicinity;
 4. _In-person participants_ will immediately comply with any requirements or
    directions made or given by Venue relating in any way to the health and
-   safety of the participants, the Organizers, sponsors and Venue personnel;
-5. _In-person participants_ should not bring food /or beverages (other than
-   bottled water or soft drinks) into the Venue which have not been authorized
-   by the Organizers in writing.
-6. _For online participants_, no attempt shall be made to gain unauthorized
+   safety of the participants, the Organisers, sponsors and Venue personnel;
+5. _In-person participants_ assume the responsibility of looking after their own personal belongings.
+<!-- 5. _In-person participants_ should not bring food /or beverages (other than
+   bottled water or soft drinks) into the Venue which have not been authorised
+   by the Organisers in writing. -->
+6. _For online participants_, no attempt shall be made to gain unauthorised
    access to the Event, and will only use the invitation link sent to them for
    their own use; sharing the invitation link to the Online Event with
-   unauthorized users can result in a ban from participation in the Event; the
-   Organizers reserves the rights to investigate and deny entry to the Event
-   should they suspect such unauthorized activities occur;
+   unauthorised users can result in a ban from participation in the Event; the
+   Organisers reserves the rights to investigate and deny entry to the Event
+   should they suspect such unauthorised activities occur;
 7. The _in-person participants_ shall not erect any stands, displays,
    advertisements, signs, notices, or other similar matters in the Venue; or fix
    any items to the walls, floor or other temporary or permanent structures of
    Venue. Similarly, the _online participants_ should not have any form of
-   advertisement on the Conference Platform, unless authorized to by the
-   Organizers.
+   advertisement on the Conference Platform, unless authorised to by the
+   Organisers.
 8. for _online participants_, no recordings of sessions or chat transcripts run
    on the Conference Platform will be created, streamed or otherwise made
-   accessible outside the Conference Platform, unless authorized by the
-   Organizers;
+   accessible outside the Conference Platform, unless authorised by the
+   Organisers;
 
 ### Minors
 
```

Comment:
Do we know what would be a customary age for CZ? For Sweden I would have no problem setting this age limit at 15.

Reply:
nope, I can ask but keeping it at 18 to be safe.